frontend/components/Site/Home/TopSection.tsx

```diff
@@ -1,25 +1,35 @@
-import SignupForm from '../SignupForm'
+import Link from 'next/link'
 import { width } from '../../../utils'
 import theme from '../../../theme'
+import Button, { ButtonSize } from '../../../elements/Button'
 
 const TopSection = () => (
   <div className="home-section-top">
     <h1>Improve your language skills and connect with others through journaling.</h1>
 
-    <SignupForm />
+    <Link href="/dashboard/signup">
+      <Button className="get-started-btn" size={ButtonSize.Large}>
+        Get started!
+      </Button>
+    </Link>
 
     <style jsx>{`
       .home-section-top {
         display: flex;
         flex-direction: column;
-        justify-content: center;
+        justify-content: space-between;
         padding: 30px 20px;
         margin: 0 auto;
         max-width: ${width.desktopHD}px;
       }
 
       h1 {
-        ${theme.typography.headingXL}
+        ${theme.typography.headingXL};
+      }
+
+      :global(.get-started-btn) {
+        white-space: nowrap;
```

Comment:
Oooo good find! I think this `white-space: nowrap;` would be great to add to the default `button { } ` styles in `Button.tsx`

Reply:
Woot! Are you thinking just as a boolean prop perhaps? Or just globally?

Reply:
just as another line in the Button.tsx styles methinks!

<img src="https://user-images.githubusercontent.com/5829188/86148576-ab0cb000-baaf-11ea-8db3-3ee13d4d8c82.png" width="400" />

Reply:
Done 🙂✅